docs/index.html

```diff
@@ -5,115 +5,169 @@
 featured_image: /images/bubble-sort.jpg
 ---
 
+<!-- TODO: Translate the Bubble Sort page into a markdown file. Too much HTML.
+---
+layout: default
+---
+<div class="home">
+  {% capture index %}{% include index.md %}{% endcapture %}
+  {{ index | markdownify }}    
+</div>
+-->
+
```

Comment:
Need to look into this more so I can convert this file into a Markdown file. Want to keep these more simple.